src/add-field-dropdown/add-field-dropdown.component.html

```diff
@@ -1,8 +1,8 @@
 <div *ngIf="!disabled" class="btn-group" dropdown keyboardNav="true" (onToggle)="focusElementIfOpen($event, expressionInput)">
-  <button id="simple-btn-keyboard-nav" type="button" class="editor-btn-add-subfield" dropdownToggle>+ <span class="caret"></span></button>
+  <button id="simple-btn-keyboard-nav" type="button" class="btn-add-field-dropdown" dropdownToggle><ng-content></ng-content> <span class="caret"></span></button>
```

Comment:
Curiosity, What is the `<ng-content></ng-content>` for?

Not sure if it is due to these changes but the styling of the dropdown in table list has some small problems:

![dropdown-filter](https://cloud.githubusercontent.com/assets/580727/21183888/4f8e6576-c20a-11e6-8621-789917e393a9.gif)

* The text inside the box should have the same dark color as the global `Add Field` you just added. Here is it very light.
* The button itself with the `+` to add a new element is too light and seems disabled. Should have the same color as that button is complex fields.